### DISCUSSION TITLE
Build error: `_fail` has package access level but no -package-name was specified

### DISCUSSION THREAD

**Discussion by krodak** [pointfreeco/swift-issue-reporting#103] [General]
Following error is reproducible in version `1.2.2` and earlier when using custom toolchain.
```
.../SourcePackages/checkouts/xctest-dynamic-overlay/Sources/IssueReporting/Unimplemented.swift:179:14: 
'_fail' has a package access level but no -package-name was specified: 
.../SourcePackages/checkouts/xctest-dynamic-overlay/Sources/IssueReporting/Unimplemented.swift
```
Toolchain used: `"com.guardsquare.ixguard"` provided by Guardsquare for RASP toolset compatibility

**Comment by mbrandonw** [comment]
Hi @krodak, thanks for the report, but it would be helpful if you could provide a project that demonstrates this problem. Since it's not clear that this is an issue with our library, I am going to convert this to a discussion. Feel free to continue the conversation over there.
